Sources/Filters.swift

```diff
@@ -89,6 +89,34 @@ struct StringFilters {
     }
   }
 
+  /// Converts camelCase to snake_case. Takes an optional Bool argument for making the string lower case,
+  /// which defaults to true
+  ///
+  /// - parameter value: the value to be processed
+  /// - parameter arguments: the arguments to the function, expecting zero or one argument
+  /// - returns: the snake case string
+  /// - throws: FilterError.invalidInputType if the value parameter isn't a string
+  static func camelToSnakeCase(_ value: Any?, arguments: [Any?]) throws -> Any? {
+    var toLower = true
+    if arguments.count ==  1 {
```

Comment:
It might be useful to refactor out this bit (to parse a boolean parameter). That way you can also simplify your tests.